tests/unit/dialog/core.js

```diff
@@ -84,6 +84,37 @@ QUnit.test( "ARIA", function( assert ) {
 	element.remove();
 } );
 
+QUnit.test( "aria-modal", function( assert ) {
+	assert.expect( 9 );
+
```

Comment:
Let's declare the variables once, that's what really happens anyway:
```suggestion

var element, wrapper;

```